app/lib/services/contact_service.dart

```diff
@@ -19,7 +20,20 @@ Future<FlutterPkid> _getPkidClient() async {
 
 Future<List<PkidContact>> getPkidContacts() async {
   FlutterPkid client = await _getPkidClient();
-  final pKidResult = await client.getPKidDoc('contacts');
+  Map<String, dynamic> pKidResult;
+  try {
+    pKidResult = await client.getPKidDoc('contacts');
+    if (pKidResult.containsKey('error')) {
+      if (pKidResult.containsValue('Keypair not found')){

```

Comment:
please update the error message to the latest

https://github.com/threefoldtech/threefold_connect_flutter_pkid_client/blob/0c1fbc34952afecbb7b77758eb1483b94c57a1ae/lib/flutter_pkid.dart#L40